instrumentation/struts-2.3/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+plugins {
+    `java-library`
+    id("io.opentelemetry.instrumentation.auto-instrumentation")
+}
+
+repositories {
+    mavenCentral()
+}
+val versions: Map<String, String> by extra
+dependencies {
+    api("io.opentelemetry.javaagent.instrumentation:opentelemetry-javaagent-servlet-3.0:${versions["opentelemetry_java_agent"]}")
```

Comment:
this can be also defined in the test scope since we are not adding any runtime classes.